apps/coordinator/ts/proof/__tests__/proof.service.test.ts

```diff
@@ -120,23 +93,6 @@ describe("ProofGeneratorService", () => {
     await expect(service.generate(defaultProofArgs)).rejects.toThrow(ErrorCodes.PRIVATE_KEY_MISMATCH.toString());
   });
 
-  test("should throw error if poll is not found", async () => {
-    const service = new ProofGeneratorService(fileService, sessionKeysService);
-
-    await expect(service.generate({ ...defaultProofArgs, poll: 2 })).rejects.toThrow(
-      ErrorCodes.POLL_NOT_FOUND.toString(),
-    );
-  });
-
-  test("should throw error if poll is not found in maci contract", async () => {
```

Comment:
Why was this test removed?

Reply:
We used this tests before implementing the sdk into the coordinator service. My first call was to just remove the test because it would be the same as testing the sdk itself.

I am trying to readd it

Reply:
I created this issue to follow up this comment #2639